crates/astria-sequencer-relayer/tests/blackbox/main.rs

```diff
@@ -273,7 +273,7 @@ async fn should_filter_rollup() {
         .await;
     sequencer_relayer
         .timeout_ms(
-            2_000,
+            10_000,
```

Comment:
This seems like it might just fix the tests itself (compared to swapping out reqwest for isahc)?

Reply:
It was more the calls to `TestSequencerRelayer::get_from_relayer_api` which were causing multiple tests to fail - specifically the 100ms timeout inside that method.

Even after fixing that (by using isahc) this test was occasionally taking just over 2 seconds on macOS.  I didn't check if we also see that on Linux.